src/uct/ib/base/ib_device.c

```diff
@@ -1125,9 +1125,10 @@ ucs_status_t uct_ib_device_mtu(const char *dev_name, uct_md_h md, int *p_mtu)
     return UCS_OK;
 }
 
-int uct_ib_device_is_gid_raw_empty(uint8_t *gid_raw)
+int uct_ib_device_is_valid_gid(const union ibv_gid *gid)
 {
-    return (*(uint64_t *)gid_raw == 0) && (*(uint64_t *)(gid_raw + 8) == 0);
+    /* Check if interface_id (lower 64 bits) is non-zero */
```

Comment:
IMO excessive commenting ("trivial" explanation for obvious & simple code..)

Reply:
removed it